cmd/maestro/servecmd/cmd.go

```diff
@@ -34,11 +35,14 @@ func runServer(cmd *cobra.Command, args []string) {
 		glog.Fatalf("Unable to initialize environment: %s", err.Error())
 	}
 
+	// Create event hub to broadcast resource status update events to subscribers
+	eventHub := event.NewEventHub()
```

Comment:
only creating when grpc server is enabled?

Reply:
Two reasons why not disable event hub optionally.
1. Event hub can work independently. When grpc server is disabled, there will be no event client registered to event hub.
2. The cloudevent source client, which receive resource status, doesn't know if the event hub is disabled or not.

I prefer to keep event hub enabled for now.